### PR TITLE
Styles for Device Lab Form

### PR DESCRIPTION
**Pull request by nicolemors**
RE: https://trello.com/c/WMM4cQ0r/264-style-device-lab-form

Added in styes for the JetPack Device Lab Form in the `.CmsContent` component.
`.comment-form` and `.pushbutton-wide` are both JetPack classes.

It should look like this:
screen shot 2016-07-08 at 4 12 16 pm

@tylersticka

### CODE REVIEW

File: src/assets/toolkit/styles/components/cms-content.css [hunk 1]
```diff
@@ -9,3 +9,138 @@
   margin-right: calc(-1 * var(--cms-content-space));
   margin-left: calc(-1 * var(--cms-content-space));
 }
+
+/**
+ * Jetpack Device Lab Form Styles
```
Comment (tylersticka) [inline src/assets/toolkit/styles/components/cms-content.css]: Can you also add to this comment chunk:

```
TODO: Use mixins to keep this stuff DRY in the future.
```

File: src/assets/toolkit/styles/components/cms-content.css [hunk 2]
```diff
@@ -9,3 +9,138 @@
...
+  border-color: var(--JetpackInput-border-color-focus);
+  outline: none;
+}
+
+/**
+ * Jetpack Form Labels
+ */
+
+:root {
+  --JetpackLabel-color: var(--color-blue);
```
Comment (tylersticka) [inline src/assets/toolkit/styles/components/cms-content.css]: Why are labels blue? I think they're blue for float labels, but default normally.

**Comment by tylersticka** [comment]
Couple of small comments inline. Resolve those and you're good to merge! Looks way better. 👍

**Comment by nicolemors** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;WMM4cQ0r&#x2F;264-style-device-lab-form"> Style device lab form</a>

**Pull request merged**